### ISSUE TITLE
[OUDS] Rename documentation page to plural if it contains a group

### ISSUE THREAD

**Issue by MaxLardenois** [Orange-OpenSource/Orange-Boosted-Bootstrap#3124]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Should be plural
- [x] Tags
- [x] Chips
- [x] Badges

Should not be plural
- [ ] Link (will be a group when Expand link is implemented)
- [ ] Button (will be a group with Pagination button and Expand button)

Filenames should also reflect this.

### Motivation and context

To implement the decided nomenclature and avoid title clash inside pages.
